.githooks/pre-commit

```diff
@@ -19,7 +19,7 @@ if [[ -n "${STAGED_GO_FILES}" ]]; then
     git add docs
 fi
 
-STAGED_EVG_FILES=$(git diff --cached --name-only | grep -e "evergreen.yml$" -e "release.yml$")
+STAGED_EVG_FILES=$(git diff --cached --name-only | grep "evergreen.yml$")
 
```

Comment:
I have noticed that when evergreen validates all included files so we don't need this change